sdk/eventhub/azure-eventhub/azure/eventhub/_producer_client.py

```diff
@@ -830,7 +890,7 @@ def get_partition_properties(self, partition_id: str) -> Dict[str, Any]:
             partition_id
         )
 
-    def flush(self, **kwargs: Any) -> None:
+    def flush(self, *, timeout: Optional[float] = None, **kwargs: Any) -> None: # pylint:disable=unused-argument
```

Comment:
oh I think the above were all for timeout also, but timeout is being used? Why is it yelling about an unused arg?

Reply:
majority of them are the unused `**kwargs`, since they were all public APIs I left them as is.